package.json

```diff
@@ -7,14 +7,14 @@
     "./src/crypto-sha1-2.js": "./src/crypto-sha1-2-browser.js"
   },
   "scripts": {
-    "test": "aegir-test",
-    "test:browser": "aegir-test browser",
+    "test": "aegir-test --webworker",
```

Comment:
aegir could use a proper cli interface yes...but don't know when to do things like that...